docs/ruleset_functions/get_zone_supply_return_exhaust_relief_terminal_fan_power_dict.md

```diff
@@ -0,0 +1,85 @@
+
+**Returns:**  
+- **get_zone_supply_return_exhaust_relief_terminal_fan_power_dict**: The function returns a dictionary that saves each zone's supply, return, exhaust, relief and terminal fan power as a list {zone.id: [supply fan power kW, return fan power kW, exhaust fan power kW, relief fan power kW, terminal fan power]}. Values will be equal to zero where not defined for a fan system.
+ 
+**Function Call:**  
+1. get_dict_of_zones_and_terminal_units_served_by_hvac_sys()   
+2. get_list_hvac_systems_associated_with_zone()   
+3. get_fan_object_electric_power()  
+4. get_fan_system_object_supply_return_exhaust_relief_total_kW_CFM()  
+
+
+## Logic:    
+- Create dictionary of hvac systems and associated zones and terminal units: `dict_of_zones_and_terminal_units_served_by_hvac_sys_x = get_dict_of_zones_and_terminal_units_served_by_hvac_sys(RMI)`  
+- For each Zone in the RMI (whichever is sent to this function): `for zone in RMI...Zone:`  
+    - Reset the zone total supply fan power variable: `zone_total_supply_fan_power = 0`
+    - Reset the zone total return fan power variable: `zone_total_return_fan_power = 0`  
+    - Reset the zone total exhaust fan power variable: `zone_total_exhaust_fan_power = 0`  
+    - Reset the zone total relief fan power variable: `zone_total_relief_fan_power = 0`
+    - Reset the zone total terminal fan power variable: `zone_total_terminal_fan_power = 0`  
+
+    - Get a list of HVAC systems serving the zone: `hvac_sys_list_serving_zone_x =  get_list_hvac_systems_associated_with_zone(RMI,zone.id)`  
+    - For each hvac system serving the zone: `for hvac in hvac_sys_list_serving_zone_x:`  
+        - Reset total_terminal_air_flow variable to 0 (total across all zones served): `total_terminal_air_flow = 0`  
+        - Reset the zone_primary_air_flow variable to 0: `zone_primary_air_flow = 0`  
+        - Check if there is a fan system associated with the hvac system (if not then assume the fan is defined at the terminal unit like it is for a four pipe fan coil units and go to Else): `if hvac.fan_system != Null:`  
+            - Get list of zones served by the hvac system: `zone_list_hvac_sys_x = dict_of_zones_and_terminal_units_served_by_hvac_sys_x[hvac.id]["ZONE_LIST"]`  
+            - Check if the hvac system serves more than one zone, if it does then carry on, if not then go to Else: `if len(zone_list_hvac_sys_x) >1:`  
+                - Get list of terminal units served by the hvac system: `terminal_list_hvac_sys_x = dict_of_zones_and_terminal_units_served_by_hvac_sys_x[hvac.id]["ZONE_LIST"]["Terminal_Unit_List"]`  
+                - For each terminal unit served by the HVAC system: `for terminal in terminal_list_hvac_sys_x:`  
+                    - Get the primary air flow: `primary_air_flow_terminal = terminal.primary_airflow`  
+                    - Add to the total cfm across all terminals served by the HVAC system: `total_terminal_air_flow = total_terminal_air_flow + primary_air_flow_terminal`  
```

Comment:
@claperle just to confirm, the total terminal airflow is the total airflow for the HVAC system, correct?

Reply:
@supriyagoel yes, exactly